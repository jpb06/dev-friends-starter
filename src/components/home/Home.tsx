import { Card, CardContent, Grid, Typography } from '@mui/material';
import Head from 'next/head';
import Image from 'next/image';

import Happy from '@public/img/happy.jpg';

export const Home = () => (
  <>
    <Head>
      <title>Index</title>
    </Head>
    <Grid container direction="row" justifyContent="center" alignItems="center">
      <Card
        sx={{
          maxWidth: 600,
          minWidth: 600,
          marginTop: 25,
        }}
      >
        <Image src={Happy} alt="Friends" priority />
        <CardContent>
          <Typography gutterBottom variant="h5" component="h2">
            My dev friends
          </Typography>
          <Typography variant="body2" color="textSecondary" component="p">
            Index
          </Typography>
        </CardContent>
      </Card>
    </Grid>
  </>
);
