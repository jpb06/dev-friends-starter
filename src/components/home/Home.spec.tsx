import { render } from '@testing-library/react';

import { Home } from './Home';

describe('Home components', () => {
  it('should match snapshot', () => {
    const { baseElement } = render(<Home />);

    expect(baseElement).toMatchSnapshot();
  });
});
