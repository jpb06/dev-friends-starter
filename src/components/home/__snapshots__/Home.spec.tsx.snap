// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home components should match snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiGrid-root MuiGrid-container css-1hbmzt3-MuiGrid-root"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1qpxjzl-MuiPaper-root-MuiCard-root"
      >
        Next image stub
        <div
          class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
        >
          <h2
            class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
          >
            My dev friends
          </h2>
          <p
            class="MuiTypography-root MuiTypography-body2 css-r40f8v-MuiTypography-root"
          >
            A new adventure begins ðŸš€
          </p>
        </div>
      </div>
    </div>
  </div>
</body>
`;
